scripts/gen.py

```diff
@@ -819,9 +824,18 @@ def generate_aten_to_xla(ctx, tree, rwxtree, fname, sig, rwsig, params, fnopts):
       xla_ref_param = param_vars[-1]
   code += tfetcher.generate_fetches()
   result_assign = generate_result_assignment(tree, _RESULT_NAME)
-  code += '  {}{};\n'.format(
-      result_assign, get_handling_function(ctx, fname, xla_ref_param,
-                                           param_vars))
+  # FIXME: This hack should be removed soon once we update aten signatures.
```

Comment:
When is this happening (removing the hack)?
Internally we use the convention of not having things like `TODO(somaname)` or `FIXME`, but we instead create an issue and we put a TODO like:
```Python
# TODO($LINK_TO_ISSUE): Blah blah blah ...
```

https://www.youtube.com/watch?v=djrrskjfYF0

😄

Reply:
This is on @smessmer 's plan to fix in his next a few PRs because it's too much for a single PR ;) Just created https://github.com/pytorch/pytorch/issues/40250 to track